diff --git a/bundles/chunk-HAJOG5B7.js b/bundles/chunk-HAJOG5B7.js
index 431ebe21ef01d7af94aae426eedd463f6d61dd5d..3b4df809877d2cd8adeb1f22efbd5d3d8ab380b9 100755
--- a/bundles/chunk-HAJOG5B7.js
+++ b/bundles/chunk-HAJOG5B7.js
@@ -8,7 +8,7 @@ import {
 } from "./chunk-GMSUYBZP.js";
 
 // bazel-out/darwin-fastbuild/bin/packages/compiler-cli/src/ngtsc/file_system/src/util.mjs
-var TS_DTS_JS_EXTENSION = /(?:\.d)?\.ts$|\.js$/;
+var TS_DTS_JS_EXTENSION = /(?:\.d)?\.tsx?$|\.jsx?$/;
 function normalizeSeparators(path) {
   return path.replace(/\\/g, "/");
 }
